Alpaca.Markets/Helpers/IntervalExtensions.cs

```diff
@@ -6,7 +6,7 @@
 public static class IntervalExtensions
 {
     /// <summary>
-    /// Gets the <see cref="Interval{DateOnly}"/> instance from to the <see cref="Interval{DateTime}"/> instance dates.
+    /// Converts a <see cref="DateTime"/> Interval into a <see cref="DateOnly"/> Interval.
```

Comment:
Please, keep generic types in the `cref` attributes - they work properly in VS and in the generated documentation. In case of your changes, the user will not able to navigate the `Interval<T>` page from the documentation.

But of course, you can re-phrase the description for better understanding :)